clients/web/src/views/layout/FooterView.js

```diff
@@ -9,9 +9,16 @@ import 'girder/stylesheets/layout/footer.styl';
  * This view shows the footer in the layout.
  */
 var LayoutFooterView = View.extend({
+
+    initialize: function (settings) {
+        var mail = settings.contactEmail; // Has to change
```

Comment:
I think that's a good idea.

This case will only apply when a some external code imports and reuses this view, but it should be handled gracefully.